apps/web/Dockerfile

```
@@ -0,0 +1,48 @@
+FROM node:18-alpine AS base
+
+# Install pnpm
+RUN npm install -g pnpm
```

Hadolint (2.14.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)